### DISCUSSION TITLE
How to convert a number to hexadecimal?

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/nuclei#4186] [Q&A] [answered]
like
188 =>  bc
c = 12
b = 11 * 16 + 12 == 188

This is not expected
```
{{hex(188)}}
```

### References
- https://github.com/projectdiscovery/nuclei/blob/main/docs/template-example/helper-functions.mdx
- https://github.com/projectdiscovery/nuclei/blob/main/docs/template-guide/helper-functions.mdx
- https://docs.nuclei.sh/template-guide/helper-functions

I consulted the above, but couldn't find anything suitable.
Did I miss something? Please take the time to tell me, thank you very much

**Comment by MetzinAround** [comment, marked as answer]
There is a helper function that converts an input number into a hex

`dec_to_hex(number number | string) string`

**Reply by hktalent** [reply]
thanks
